lib/SILOptimizer/LoopTransforms/ArrayPropertyOpt.cpp

```diff
@@ -107,26 +150,55 @@ class ArrayPropertiesAnalysis {
     // beneficial. This heuristic also simplifies which regions we want to
     // specialize on. We will specialize the outermost loopnest that has
     // 'array.props' instructions in its preheader.
+
```

Comment:
Did you do this reformatting on purpose? If yes, you might want to put it into a separate commit, e.g. with a message "Improve formatting [...] NFC". Just to make it clear.

Reply:
This wasn't on purpose. I'll get rid of it. Its left-over from undoing the clang-format.

Reply:
LLVM style is to omit curly brackets for single line conditional clauses. I have a `git-format` script that does this:
`git clang-format --binary ${CLANG_FORMAT} --style "{BasedOnStyle: LLVM, BreakBeforeBinaryOperators: NonAssignment}"`

The break-before-binary operators is something I do to avoid bugs in compound multiline expressions where, if the operator is hidden at the end of the line in some arbitrary column amidst the noise, my brain interpolates the operator I expect so otherwise I never spot those bugs.

Reply:
@atrick if we want to use that style, we should add it to Swift's default clang-format.

Reply:
@gottesmm The last discussion about changing the default ended with no one opposing this variation in style, most people agreeing that this was within the current guidelines, but almost everyone opposing having a discussion about changing the default. For me, it's a very real issue of the quality of the code I work on. It's got nothing to do with making things pretty.

Reply:
That isn't what I meant. I meant put up a PR. I am fine with the change = ).

Reply:
clang-format is reordering more than what I changed. Are such changes generally ok ?